utils/config.go

```diff
@@ -825,3 +831,11 @@ func (cc *configContext) setChainConfig() (err error) {
 	cc.cfg.chainCfg, err = getChainConfig(cc.cfg.ChainID, "")
 	return err
 }
+
+// ToTitleCase adjusts fork names to title case
```

Comment:
nit: I would mention that if there is a word 'glacier' anywhere in the string, thats also transformed to `TitleCase`